package.json

```diff
@@ -15,6 +15,7 @@
     "test:integration": "ng test --project integration --runInBand",
     "test:integration:watch": "ng test --project integration --runInBand --watch",
     "build": "ng-packagr -p src/package.json && cpx README.md dist",
```

Comment:
Add the `build:migration` command to the `build` command.

Reply:
done!